package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "fiware-sth-rest-api",
```

Comment:
Assuming I'm not an expert in NodeJs nor javascript :) This `package.json` seems to be a container describing a component, in this case called `fiware-sth-rst-api`, right? There will be in the future another `package.json` file describing another component? If so, how will you differentiate among them? I mean, I'm not seeing any specific folder under `src/` nor any folder structure for packages e.g. `com.telefonica.iot.fiware-sth-rest-api` (I'm thinking on Java terms here :)).